pipeline/src/default_Rt.jl

```diff
@@ -0,0 +1,20 @@
+"""
+Compute the default Rt values over time.
+
+## keyword Arguments
+- `A`: Amplitude of the sinusoidal variation in Rt. Default is 0.3.
+- `P`: Period of the sinusoidal variation in Rt. Default is 30 time steps.
+
+## Returns
+- `true_Rt`: Array of default Rt values over time.
+
+"""
+function default_Rt(; A = 0.3, P = 30.0)
```

Comment:
under the type dispatch idea this would be come i.e `generate_Rt(<type>; stuff)`.